llmfoundry/data/text_data.py

```diff
@@ -328,9 +324,11 @@ def build_text_dataloader(
         if 'streams' in dataset_cfg else None,
     )
 
-    valid_streaming_text_dataset_parameters = inspect.signature(
-        StreamingTextDataset,
-    ).parameters
+    valid_streaming_text_dataset_parameters = set(
```

Comment:
this code turns this line and the next one into a noop right?

Reply:
100%. logic is further cleaned up. plz tal.